### DISCUSSION TITLE
Homebridge LG ThinQ Plugin Config

### DISCUSSION THREAD

**Discussion by cwmn** [nVuln/homebridge-lg-thinq#218] [Q&A]
Newbie, using the latest version of Hombridge and the LG plugin. I put in my username and password, I've tried tokens. The log file says the connection to LG is successful yet a few lines later it says "No ThinQ devices in your account". The LG ThinQ app shows my washer and dryer and all of their associated information. I am assuming that the device information should pull from LG or do I need to fill in the device information in the Homebridge plugin configuration? Any help is appreciated.

**Comment by nVuln** [comment]
go to plugin setting and see if all your devices show up, then press `Save` button, like this

<img width="827" alt="Screenshot 2023-02-28 at 4 14 00 PM" src="https://user-images.githubusercontent.com/54855446/221807209-17997b42-b454-4b2c-a6dc-d752338b23ef.png">

**Comment by cwmn** [comment]
No my devices don’t show in the plug-in yet the log file says that it is connected to LGSent from my iPhoneOn Feb 28, 2023, at 3:15 AM, nVuln ***@***.***> wrote:﻿
go to plugin setting and see if all your devices show up, then press Save button, like this

**Reply by nVuln** [reply]
Make sure you signing to same account on lg app, please re-check
